Makefile

```diff
@@ -225,13 +225,18 @@ delete-tag:
 docker-otelcol:
 	COMPONENT=otelcol $(MAKE) docker-component
 
+# build collector binaries with different OS and Architecture
 .PHONY: binaries-all-sys
-binaries-all-sys: binaries-darwin_amd64 binaries-linux_amd64 binaries-linux_arm64 binaries-windows_amd64
+binaries-all-sys: binaries-darwin_amd64 binaries-linux_amd64 binaries-linux_arm64 binaries-windows_amd64 binaries-darwin_arm64
```

Comment:
Can you move this as the second in the list to have things ordered?

Reply:
Done!